docs/GHA.md

```diff
@@ -12,17 +12,3 @@ In a GitHub Actions workflow, you can set these environmental variables using th
       core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
       core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
 ```
-
-To write to the cache, set `SCCACHE_GHA_CACHE_TO` to a cache key, for example
-`sccache-latest`. To read from cache key prefixes, set `SCCACHE_GHA_CACHE_FROM`
-to a comma-separated list of cache key prefixes, for example `sccache-`.
-
-In contrast to the [`@actions/cache`](https://github.com/actions/cache) action, which saves a single large archive per cache key, `sccache` with GHA cache storage saves each cache entry separately.
```

Comment:
This paragraph was added to explain why sccache will create many small caches instead of the usual single large caches. https://github.com/mozilla/sccache/pull/1433#issuecomment-1333478389

Reply:
I think our users don't need to care about those service internal details.

Reply:
well, it isn't really internal as it shows on github 

I am trying with another project and I can see caches of several gb to a few kb
https://github.com/sylvestre/coreutils/actions/caches

Reply:
I agree with @jakelee8 - documenting visible behavior is a must.

Reply:
Should be addressed by https://github.com/mozilla/sccache/pull/1533